### DISCUSSION TITLE
ht3d heat transfer

### DISCUSSION THREAD

**Discussion by Matteo-Frassinelli** [firemodels/fds#12745] [Q&A]
Hi everyone,
this is my first post, so a short introduction: my name is Matteo and I'm a fire safety engineer at ESA Engineering (Florence, IT), my main occupation is FSE analysis.
I'm currently trying to improve my analyzes with heat conduction in solids, using pyrosim version 2023.3.1312 and fds version 6.8.0.
I reproduced the "ht3d_ibeam" example from the validation section and it works fine.
Following the fds user's guide, I'm now trying to modify this example by replacing the heat source (on the solid's surface) with a burner under the beam but I can't get the heat transfer inside the solid, something isn't working.

![Screen](https://github.com/firemodels/fds/assets/166003409/4c2bfd7f-3d34-4709-a7ee-38a03c5c1965)

In addition to the new burner I activated the gas phase and the solid's surfaces are 3D.

Can anyone help me or direct me to useful examples to solve my problem?

Below my fds script. 

Thank you very much

Best regards

ht3d_ibeam_burner.fds
Generated by PyroSim 2023.3.1312
4 apr 2024, 14:36:05

-------------User Section (not generated by PyroSim)-------------

&BNDF QUANTITY='WALL TEMPERATURE', CELL_CENTERED=T/
&BNDF QUANTITY='NET HEAT FLUX', CELL_CENTERED=T/

--------------------PyroSim-generated Section--------------------

&HEAD CHID='ht3d_ibeam_burner', TITLE='Compare 3D heat transfer with ANSYS model'/
&TIME T_END=3600.0, DT=0.5, WALL_INCREMENT=1/
&DUMP DT_DEVC=10.0/
&MISC SOLID_PHASE_ONLY=.FALSE./

&MESH ID='Mesh01', IJK=44,44,57, XB=-0.22,0.22,-0.22,0.22,-0.13,0.44/

&REAC ID='Reaction1',
      FYI='Methanol C_1 H_4 O_1',
      FUEL='METHANOL',
      CO_YIELD=1.0E-3,
      SOOT_YIELD=1.0E-3/

&DEVC ID='TS_x195-40', QUANTITY='WALL TEMPERATURE', XYZ=0.195,0.195,0.081, IOR=3, TIME_AVERAGED=.FALSE./
&DEVC ID='TS_x145-30', QUANTITY='WALL TEMPERATURE', XYZ=0.145,0.095,0.081, IOR=3, TIME_AVERAGED=.FALSE./
&DEVC ID='TS_x095-20', QUANTITY='WALL TEMPERATURE', XYZ=0.095,-5.0E-3,0.081, IOR=3, TIME_AVERAGED=.FALSE./
&DEVC ID='TS_x025-40', QUANTITY='WALL TEMPERATURE', XYZ=0.025,0.195,0.081, IOR=3, TIME_AVERAGED=.FALSE./
&DEVC ID='TS_x195-01', QUANTITY='WALL TEMPERATURE', XYZ=0.195,-0.195,0.081, IOR=3, TIME_AVERAGED=.FALSE./
&DEVC ID='TS_x025-01', QUANTITY='WALL TEMPERATURE', XYZ=0.025,-0.195,0.081, IOR=3, TIME_AVERAGED=.FALSE./

&MATL ID='STEEL',
      SPECIFIC_HEAT=0.6,
      CONDUCTIVITY=45.0,
      DENSITY=7850.0/

&SURF ID='ADIABATIC STEEL',
      COLOR='GRAY',
      HEAT_TRANSFER_COEFFICIENT=0.0,
      MATL_ID(1,1)='STEEL',
      MATL_MASS_FRACTION(1,1)=1.0,
      HT3D=.TRUE.,
      CELL_SIZE=0.01/
&SURF ID='FIRE',
      COLOR='RED',
      HRRPUA=100.0/

&OBST ID='bot flange', XB=-0.2,0.2,-0.2,0.2,0.02,0.08, SURF_ID='ADIABATIC STEEL'/ 
&OBST ID='top flange', XB=-0.2,0.2,-0.2,0.2,0.36,0.42, SURF_ID='ADIABATIC STEEL'/ 
&OBST ID='web', XB=-0.02,0.02,-0.2,0.2,0.08,0.36, SURF_ID='ADIABATIC STEEL'/ 

&VENT ID='Mesh Vent: Mesh01 [XMAX]', SURF_ID='OPEN', XB=0.22,0.22,-0.22,0.22,-0.13,0.44/ 
&VENT ID='Mesh Vent: Mesh01 [XMIN]', SURF_ID='OPEN', XB=-0.22,-0.22,-0.22,0.22,-0.13,0.44/ 
&VENT ID='Mesh Vent: Mesh01 [YMAX]', SURF_ID='OPEN', XB=-0.22,0.22,0.22,0.22,-0.13,0.44/ 
&VENT ID='Mesh Vent: Mesh01 [YMIN]', SURF_ID='OPEN', XB=-0.22,0.22,-0.22,-0.22,-0.13,0.44/ 
&VENT ID='Mesh Vent: Mesh01 [ZMAX]', SURF_ID='OPEN', XB=-0.22,0.22,-0.22,0.22,0.44,0.44/ 
&VENT ID='Vent #6', SURF_ID='FIRE', XB=-0.05,0.05,-0.05,0.05,-0.13,-0.13/ 

&BNDF QUANTITY='WALL TEMPERATURE'/
&BNDF QUANTITY='NET HEAT FLUX'/

&SLCF QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE., ID='Y = 0,1 m: Temperature', PBY=0.1/
&SLCF QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE., ID='Y = 0,0 m: Temperature', PBY=0.0/
&SLCF QUANTITY='TEMPERATURE', CELL_CENTERED=.TRUE., ID='Y = -0,1 m: Temperature', PBY=-0.1/

&TAIL /

**Comment by rmcdermo** [comment]
@mcgratta Has the TMP array been populated with the average solid cell temperatures in the latest source?

This is something that used to work in the old HT3D and is convenient for visualization with slices.

**Comment by mcgratta** [comment]
I am running the case.

**Comment by mcgratta** [comment]
```
HEAT_TRANSFER_COEFFICIENT=0.0
```
This is part of the problem. You have effectively stopped heat transfer from gas to solid.

**Comment by mcgratta** [comment]
![image](https://github.com/firemodels/fds/assets/11333911/251fb412-4e2b-4e81-ba5a-2b03ddd4f25c)

The case works when I remove the HTC=0 and run with 6.9.0. It might work with 6.8.0, but I recommend you upgrade to the latest because HT3D is still being tested.

**Reply by Matteo-Frassinelli** [reply]
Hi,
first of all I apologize for the late reply, it wasn't possible for me before.
I'll follow your advice!
Thank you very much
